src/components/pagination-selector/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./style.css";
+
+export const PaginationSelector = ({ setUsersPerPage, setCurrentPage }) => {
```

Comment:
```suggestion
interface PaginationSelectorProps {
  onItemsPerPageChange: (itemsPerPage: number) => void;
}
export const PaginationSelector: React.FC<PaginationSelectorProps>  = ({ onItemsPerPageChange }) => {
```
alguns pontos aqui:
- tipagem! ajuda mto quem for usar o componente (inclusive vc no futuro 😅 )
- props que são eventos tem o prefixo `on`. nesse caso, ao invés de `setUsersPerPage`, seria melhor `onUsersPerPageChange` ou `onItemsPerPageChange`
- como esse componente está mais genérico, talvez valha nomear as props de forma mais genérica também (`onItemsChange` ao invés de `onUserPerPageChange`)
qualquer coisa chama no slack! 👍

Reply:
Muito obrigado Beto!
Essas boas práticas estou pegando aos poucos, e esses comentários/sugestões estão ajudando bastante!
Corrigido, e obrigado novamente!